includes/Classifai/Services/LanguageProcessing.php

```diff
@@ -167,6 +186,34 @@ public function generate_post_excerpt( WP_REST_Request $request ) {
 		return rest_ensure_response( $provider->rest_endpoint_callback( $post_id, 'excerpt' ) );
 	}
 
+	/**
+	 * Generates text to speech for a post using REST.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return boolean
+	 */
+	public function synthesize_speech_from_text( $request ) {
+		$post_id           = $request->get_param( 'id' );
+		$save_post_handler = new SavePostHandler();
+		$attachment_id     = $save_post_handler->synthesize_speech( $post_id );
+
+		if ( is_wp_error( $attachment_id ) ) {
+			$error_code    = $attachment_id->get_error_code();
+			$error_message = $attachment_id->get_error_message();
+
+			return array(
+				'success' => false,
+				'code'    => $error_code,
+				'message' => $error_message,
```

Comment:
Can simplify this by removing the lines above:
```suggestion
				'code'    => $attachment_id->get_error_code(),
				'message' => $attachment_id->get_error_message(),
```